workflows/staramr.nf

```diff
@@ -54,7 +54,24 @@ workflow STARAMR {
 
     // Create a new channel of metadata from a sample sheet
     // NB: `input` corresponds to `params.input` and associated sample sheet schema
+
     ch_input = Channel.fromSamplesheet("input")
+        .map { meta, contigs ->
+            // Remove characters from meta.irida_id that could cause issues in processes
+            meta.irida_id = meta.irida_id.replaceAll(/[;\\#><|]/, '_')
```

Comment:
It's probably not the *best* solution -- as you rightly suggest it could in theory cause some headaches. I wanted to avoid a chance of code injection in my bash command that uses `meta.irida_id`. In the case of `staramrnf` because we modify the `meta.irida_id` before any of the files to be included in the `iridanext.output.json.gz` it doesn't affect the `iridanext.output.json.gz`. As to running on IRIDA-Next *in theory* it shouldn't be a problem because the it seems these characters are not used in the IRIDA-Next ID [see here](https://github.com/phac-nml/irida-next/blob/21c4f731cf837e238134c067cb88b87bf0392181/lib/irida/persistent_unique_id.rb#L38-L41) but I will need to confirm this 100% before I am done with this PR.

Reply:
Reverted the change here 1d829b3